buildenv/jenkins/jobs/infrastructure/wrapper_variables.yml

```diff
@@ -149,7 +149,7 @@ PullRequest-OpenJDK:
         pull_request_builder:
             trigger_phrase: .*\bjenkins\s+(compile|test)\b.*
             cron: "H/5 * * * *"
-            admin_list: "Extentions"
+            admin_list: "Extensions"
             blacklist: "kenrai123"
```

Comment:
I've always wondered why this user was blacklisted?

Reply:
I've explained it privately.